dlt/common/runtime/anon_tracker.py

```diff
@@ -16,7 +16,7 @@
 
 from dlt.version import __version__
 
-TEventCategory = Literal["pipeline", "command", "helper"]
+TEventCategory = Literal["pipeline", "command", "helper", "data_access"]
```

Comment:
OK so we moved the event to `pipeline` so we do not need additional category. we just need additional name. ie. `access_dataset`